src/components/admin-lte/index.js

```diff
@@ -239,6 +244,7 @@ class AdminLTEPage extends React.Component {
           const textValue = childrens[i].children[0].children[1].textContent
           childrens[i].id = textValue
           const ignore = ignoreItems.find(val => textValue === val)
+          if (textValue === 'Send/Receive XEC') textValue = 'Send/Receive BCH'
```

Comment:
If you are changing textValue, better define in with `let textValue = ...` on line 244